lib/omniauth/strategies/facebook.rb

```diff
@@ -9,18 +9,15 @@ module Strategies
     class Facebook < OmniAuth::Strategies::OAuth2
       class NoAuthorizationCodeError < StandardError; end
 
-      DEFAULT_SCOPE = 'email'
+      DEFAULT_SCOPE = 'email'.freeze
+      API_VERSION = 'v2.6'.freeze
 
       option :client_options, {
-        :site => 'https://graph.facebook.com',
+        :site => "https://graph.facebook.com/#{API_VERSION}",
         :authorize_url => "https://www.facebook.com/dialog/oauth",
```

Comment:
@printercu I think we should also use the version in this URL, if I am reading the FB docs correctly. Could you make that change, if it seems right?